src/groovy/heim/session/SessionService.groovy

```diff
@@ -24,6 +24,16 @@ import java.util.concurrent.ConcurrentMap
 @Log4j
 class SessionService {
 
+
+    def SessionService() {
+        new Timer().schedule({
+            garbageCollection()
+        } as TimerTask, collectionInterval, collectionInterval)
+    }
```

Comment:
Constructors shouldn't have side effects. Move this into a method annotated with `@PostConstruct`.